frontend/src/views/KontoOverview/GddSend.vue

```diff
@@ -255,6 +255,7 @@ export default {
       const result = await communityAPI.send(
         this.$store.state.session_id,
         this.ajaxCreateData.email,
+        // better to send the user's input here?
         this.ajaxCreateData.amount * 10000,
```

Comment:
This is important. I would prefer to send `3.14` instead of `31400`

Reply:
erledigt